cli/src/lib/libDefs.js

```diff
@@ -168,6 +168,8 @@ export async function getLibDefs(defsDir: string): Promise<Array<LibDef>> {
   const defsDirItems = await fs.readdir(defsDir);
   await P.all(
     defsDirItems.map(async item => {
+      if (item === '.DS_Store') return;
```

Comment:
Could we instead get rid of the DS_Store files and add a test to make sure they don't come back?

Reply:
How do we do that? Mine randomly appeared because I opened my finder, it's not like you can gitignore it because the issue is when we run it locally and because it creates a `DS_Store` in every single dir means that when cleaning up I was trying to go into every `definitions/npm/@scope` to delete this.

Unless you're saying when we run this loop we should delete it instead of returning? Not sure which approach is best here, I imagine deleting isn't good because `DS_Store` is used to retain folder positioning if someone does open the repo in their finder which has some usage.

Reply:
Right yeah makes sense, I see how it can be an issue when running it locally. 👍🏼

Reply:
Actually I can see how this code makes no sense when reading this later. I'll add a comment to describe why this issue needs to be exited before I merge